scripts/docker/build/arangodb-build.Dockerfile

```diff
@@ -13,6 +13,9 @@ COPY install-openldap.sh /tools/
 RUN [ "/tools/install-openldap.sh", "3.1.2" ]
 RUN apk --no-cache del perl
 
+COPY install-cppcheck.sh /tools/
+RUN [ "/tools/install-cppcheck.sh", "2.11" , "2.11.0" ]
```

Comment:
```
COPY install-cppcheck.sh /tools/
RUN if [ "$(apk --print-arch)" = "x86_64" ] ; then /tools/install-cppcheck.sh 2.11 2.11.0 ; fi
```

Reply:
@KVS85 Ok, I removed it from arm images
The reason why I didn't do a check before, was to have a consistent set of tools on both architectures